deploy/central/tasks-chart/templates/task-ods-build-go-with-sidecar.yaml

```diff
@@ -8,23 +8,23 @@ spec:
   description: |-
     Builds Go (module) applications.
 
-    The following steps are executed:
-
-    - check proper formatting against gofmt
-    - linting using golangci-lint
-    - build (using `go-os` and `go-arch` parameters)
-    - test execution
-
-    Tests exclude the vendor directory. Test results are converted into xUnit format.
-
-    Both xUnit report and coverage report are placed into .ods/artifacts.
-
-    The built binary is named `app` and placed into the directory specified by `output-dir`.
-
     The exact build recipe can be found at
     link:https://github.com/opendevstack/ods-pipeline/blob/master/build/package/scripts/build-go.sh[build/package/scripts/build-go.sh].
 
-    After tests ran successfully, the application source code is scanned by SonarQube.
+    The following provides an overview of the performed steps:
+
+    - Source files are checked to be formatted with `gofmt`.
+    - `golanci-lint` is run, generating a report which is placed into
+      `.ods/artifacts/lint-reports`. The linter can be configured via a
+      config file as described in the
+      link:https://golangci-lint.run/usage/configuration/[configuration documentation].
+    - Tests are executed. A potential `vendor` directory is excluded. Test
```

Comment:
Do we want to mention that tests are only executed if no artifacts are present already?

Reply:
Thanks - added in c7aa5a0.